testdata/cheats/Vm.sol

```diff
@@ -44,6 +44,7 @@ interface Vm {
     function assertApproxEqRel(uint256 left, uint256 right, uint256 maxPercentDelta, string calldata error) external pure;
     function assertApproxEqRel(int256 left, int256 right, uint256 maxPercentDelta) external pure;
     function assertApproxEqRel(int256 left, int256 right, uint256 maxPercentDelta, string calldata error) external pure;
+    function assertContains(string calldata haystack, string calldata needle) external returns (bool result);
```

Comment:
nit: for consistency I would prefer using the `left` / `right` naming here

Reply:
Should be good now :)

Reply:
ah sorry @zerosnacks saw your comment after leaving this one https://github.com/foundry-rs/foundry/pull/9085#discussion_r1795591001